vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+    "crons": [
+      {
+        "path": "/api/cron/attendance",
+        "schedule": "0 0 * * *"
```

Comment:
This is different from what we've in the route?

Reply:
fixed